### ISSUE TITLE
`SlevomatCodingStandard.Functions.RequireMultiLineCall` incorrectly detects line length

### ISSUE THREAD

**Issue by roslov** [slevomat/coding-standard#1816]
Slevomat version: 8.25.0

Sniff: `SlevomatCodingStandard.Functions.RequireMultiLineCall`

There is a line in a legacy code:
```
        return !preg_match("/$regex/", (string) $value) ? "недопустимый формат, поле '$name'" : null;
```

The real line length is 101 chars.

<img width="1137" height="97" alt="Image" src="https://github.com/user-attachments/assets/a4da045f-2518-4319-8965-bd281c965ba6" />

<img width="779" height="526" alt="Image" src="https://github.com/user-attachments/assets/ef454033-2530-4373-9859-9cda560ab2a6" />

But the sniff calculates it as 123.

<img width="1485" height="600" alt="Image" src="https://github.com/user-attachments/assets/72d38acb-a6dc-4f5b-b9a8-33337daf3ddf" />

Probably, we need to replace `strlen` with `mb_strlen`.
